http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BuildHelmChartValues.scala

```diff
@@ -380,6 +382,26 @@ private[leonardo] object BuildHelmChartValues {
       raw"""ingress.tls[0].hosts[0]=${k8sProxyHostString}"""
     )
 
+    val autopilotParams = autopilot match {
+      case Some(v) =>
+        val ls = List(
+          raw"""autopilot.enabled=true""",
+          raw"""autopilot.app.cpu=${v.cpuInMillicores}m""",
+          raw"""autopilot.app.memory=${v.memoryInGb}Gi""",
+          raw"""autopilot.app.ephemeral\-storage=${v.ephemeralStorageInGb}Gi""",
+          raw"""autopilot.welder.cpu=${config.clusterConfig.autopilotConfig.welder.cpuInMillicores}m""",
+          raw"""autopilot.welder.memory=${config.clusterConfig.autopilotConfig.welder.memoryInGb}Gi""",
+          raw"""autopilot.welder.ephemeral\-storage=${config.clusterConfig.autopilotConfig.welder.ephemeralStorageInGb}Gi""",
+          raw"""autopilot.wondershaper.cpu=${config.clusterConfig.autopilotConfig.wondershaper.cpuInMillicores}m""",
+          raw"""autopilot.wondershaper.memory=${config.clusterConfig.autopilotConfig.wondershaper.memoryInGb}Gi""",
+          raw"""autopilot.wondershaper.ephemeral\-storage=${config.clusterConfig.autopilotConfig.wondershaper.ephemeralStorageInGb}Gi"""
+        )
+        if (v.computeClass == ComputeClass.GeneralPurpose)
+          ls
+        else raw"""nodeSelector.cloud\.google\.com/compute-class=${v.computeClass.toString}""" :: ls
```

Comment:
when it's general purpose, GCP doesn't allow us to pass the compute class value...the API behaves in a way that when the value isn't specified, general-purpose is used

Reply:
Can you add that as a comment in the code? That will help future dev understand better I think